tests/x11/gnomecase/change_password.pm

```diff
@@ -61,7 +62,18 @@ sub reboot_system {
 sub switch_user {
     assert_and_click "system-indicator";
     assert_and_click "user-logout-sector";
-    assert_and_click "switch-user";
+    assert_screen([qw(switch-user logout-user)]);
+    if (match_has_tag('switch-user'))
+    {
+        assert_and_click "switch-user";
+    }
+    else
+    {
+        record_soft_failure 'switch user disappeared after logout';
+        assert_and_click "logout-user";
+        send_key 'tab';
+        send_key 'ret';
+    }
```

Comment:
There is a bug fixed for "switcher user" in Tumbleweed. Reference [bsc#1100041](https://bugzilla.opensuse.org/show_bug.cgi?id=1100041)